.github/workflows/reformat_yaml_files.yml

```diff
@@ -0,0 +1,18 @@
+name: Reformat Yaml Files
+
+on: [push]
+
+jobs:
+  reformat_yaml_files:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      # Run the reformat action
+      - name: Reformat YAML Files
+        uses: credfeto/action-yaml-format@v1.3.2
+      - name: Commit files
+        run:  git commit --all -m"Reformat YAML Files to common format." || true
+      - name: Push
+        run: git push "https://${{github.actor}}:${{secrets.SOURCE_PUSH_TOKEN}}@github.com/${{github.repository}}.git" "HEAD:${{ env.GIT_BRANCH }}"
```

Comment:
IMHO, this is a bad practice to commit PR directly in the PR. BTW, this probably won’t be possible to commit in a PR that was created from a forked repository.

It would be better to raise an error if the YAML file isn’t right formatted.